src/components/VizbeeMiniCastBar/helpers.js

```diff
@@ -0,0 +1,69 @@
+
+/**
+   * Gets the miniplayer title to be displayed
+   *
+   * @returns {string} title to display
+   */
+export const getTitleFromVideoInfo = ({title, castingTo}) => {
+    if (title && title?.length) {
```

Comment:
?. is optional chaining operator is already used here. So no need to check `title`. Please change `title && title?.length` to `title?.length`

Reply:
Done